sim/common/cata/enchant_effects.go

```diff
@@ -380,6 +380,63 @@ func init() {
 		character.ItemSwap.RegisterOnSwapItemForEnchantEffect(4115, aura)
 	})
 
+	// Enchant: 4116, Spell: 75175 - Darkglow Embroidery
+	core.NewEnchantEffect(4116, func(agent core.Agent) {
+		character := agent.GetCharacter()
+
+		statAura := character.NewTemporaryStatsAura(
+			"Darkglow Embroidery Cata",
+			core.ActionID{SpellID: 75175},
+			stats.Stats{stats.Spirit: 480},
+			time.Second*15,
+		)
+
+		aura := core.MakeProcTriggerAura(&character.Unit, core.ProcTrigger{
+			Name:       "Darkglow Embroidery Cata",
+			ActionID:   core.ActionID{SpellID: 75171},
+			Callback:   core.CallbackOnSpellHitDealt | core.CallbackOnPeriodicDamageDealt | core.CallbackOnHealDealt,
+			ProcMask:   core.ProcMaskSpellDamage | core.ProcMaskSpellHealing,
+			Outcome:    core.OutcomeLanded,
+			ICD:        time.Second * 60,
+			ProcChance: 0.35,
```

Comment:
It's 57 sec ICD, 30% proc https://www.wowhead.com/cata/spell=75174/darkglow-embroidery

Reply:
pushed changes